web_client/.eslintcache

```diff
@@ -0,0 +1 @@
unt":0,"fixableWarningCount":0},{"filePath":"22","messages":"23","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"24","messages":"25","errorCount":0,"warningCount":3,"fixableErrorCount":0,"fixableWarningCount":2,"source":null},"/Users/justinchi/projects/Citizen-Center/web_client/store/index.js",[],"/Users/justinchi/projects/Citizen-Center/web_client/layouts/default.vue",[],"/Users/justinchi/projects/Citizen-Center/web_client/pages/index.vue",["26","27","28","29","30","31","32","33"],"/Users/justinchi/projects/Citizen-Center/web_client/components/Header.vue",[],"/Users/justinchi/projects/Citizen-Center/web_client/components/index/Item.vue",[],"/Users/justinchi/projects/Citizen-Center/web_client/components/index/SectionLinks.vue",["34","35","36"],{"ruleId":"37","severity":1,"message":"38","line":27,"column":21,"nodeType":"39","endLine":27,"endColumn":39,"fix":"40"},{"ruleId":"37","severity":1,"message":"41","line":28,"column":21,"nodeType":"39","endLine":28,"endColumn":42,"fix":"42"},{"ruleId":"37","severity":1,"message":"38","line":59,"column":21,"nodeType":"39","endLine":59,"endColumn":44,"fix":"43"},{"ruleId":"37","severity":1,"message":"41","line":60,"column":21,"nodeType":"39","endLine":60,"endColumn":51,"fix":"44"},{"ruleId":"37","severity":1,"message":"38","line":83,"column":21,"nodeType":"39","endLine":83,"endColumn":46,"fix":"45"},{"ruleId":"37","severity":1,"message":"41","line":84,"column":21,"nodeType":"39","endLine":84,"endColumn":48,"fix":"46"},{"ruleId":"37","severity":1,"message":"38","line":118,"column":21,"nodeType":"39","endLine":118,"endColumn":46,"fix":"47"},{"ruleId":"37","severity":1,"message":"41","line":119,"column":21,"nodeType":"39","endLine":119,"endColumn":39,"fix":"48"},{"ruleId":"49","severity":1,"message":"50","line":19,"column":21,"nodeType":"51","endLine":19,"endColumn":42,"fix":"52"},{"ruleId":"49","severity":1,"message":"53","line":21,"column":21,"nodeType":"51","endLine":21,"endColumn":37,"fix":"54"},{"ruleId":"55","severity":1,"message":"56","line":48,"column":9,"nodeType":"57","endLine":48,"endColumn":21},"vue/attribute-hyphenation","Attribute 'imgSrc' must be hyphenated.","VIdentifier",{"range":"58","text":"59"},"Attribute 'imgAlt' must be hyphenated.",{"range":"60","text":"61"},{"range":"62","text":"59"},{"range":"63","text":"61"},{"range":"64","text":"59"},{"range":"65","text":"61"},{"range":"66","text":"59"},{"range":"67","text":"61"},"vue/attributes-order","Attribute \"v-for\" should go before \"class\".","VDirectiveKey",{"range":"68","text":"69"},"Attribute \":key\" should go before \":link\".",{"range":"70","text":"71"},"vue/require-default-prop","Prop 'links' requires default value to be set.","Property",[1329,1335],"img-src",[1368,1374],"img-alt",[2608,2614],[2652,2658],[3545,3551],[3591,3597],[4829,4835],[4875,4881],[611,665],"v-for=\"link in links\"\n                    class=\"mt-2\"",[686,740],":key=\"link.text\"\n                    :link=\"link.href\""]
```

Comment:
woops, this should not have been checked in. i can create another PR adding it to gitignore and untrack this.